### ISSUE TITLE
Scaffold/guide to do LIKHA Collection #1 (iOS)

### ISSUE THREAD

**Issue by masarapmabuhay** [usbong/LIKHA#6]
Salut! I wondered if you're available to create a scaffold/guide to do something like [LIKHA Collection #1 ](https://play.google.com/store/apps/details?id=usbong.android.likha_collection_1&hl=en)for the iOS.

I think this is one way we can get around with the minimum fee of 0.99USD for each app on the App Store, by bundling storybooks or segments together in one 0.99USD app.

If you're willing to do this, please never hesitate to send me privately a Gantt Chart, so I can compensate you accordingly.

Merci beaucoup toujours!

**Comment by chrisamanse** [comment]
Yes, LIKHA Collection app is currently in my task list. I sent you an email regarding my thoughts on the app and its implementation.
